x/rollup/keeper/deposits.go

```diff
@@ -24,21 +23,12 @@ func (k *Keeper) setL1BlockInfo(ctx sdk.Context, info derive.L1BlockInfo) error
 	if err != nil {
 		return types.WrapError(err, "marshal L1 block info")
 	}
-	if err = k.storeService.OpenKVStore(ctx).Set([]byte(types.KeyL1BlockInfo), infoBytes); err != nil {
-		return types.WrapError(err, "set")
+	store := k.storeService.OpenKVStore(ctx)
+	if err = store.Set([]byte(types.KeyL1BlockInfo), infoBytes); err != nil {
+		return types.WrapError(err, "set latest L1 block info")
 	}
-	return nil
-}
-
-// TODO: include the logic to also store the L1 block info by blockhash in setL1BlockInfo and remove setL1BlockHistory
-// setL1BlockHistory sets the L1 block info to the app state, with the key being the blockhash, so we can look it up easily later.
-func (k *Keeper) setL1BlockHistory(ctx context.Context, info *derive.L1BlockInfo) error {
-	infoBytes, err := json.Marshal(info)
-	if err != nil {
-		return types.WrapError(err, "marshal L1 block info")
-	}
-	if err = k.storeService.OpenKVStore(ctx).Set(info.BlockHash.Bytes(), infoBytes); err != nil {
-		return types.WrapError(err, "set")
+	if err = store.Set(info.BlockHash.Bytes(), infoBytes); err != nil {
+		return types.WrapError(err, "set historical L1 block info")
```

Comment:
We no longer want to store the L1 block info by block hash in the rollup module store

```suggestion
```

Reply:
I removed this but didn't commit🤣
Thanks!🙏